packages/terra-time-input/src/orion-fusion-theme/TimeInput.module.scss

```diff
@@ -0,0 +1,199 @@
+    --terra-time-input-focus-ltr-box-shadow: 0 0 8px #a6d9f4;
+    --terra-time-input-focus-rtl-box-shadow: 0 0 8px #a6d9f4;
+    --terra-time-input-focus-color: #1c1f21;
+    --terra-time-input-focus-opacity: 1;
+
+    // Meridiem Button
+    --terra-time-input-meridiem-button-font-size: 1rem;
+    --terra-time-input-meridiem-button-background-color: #fff;
+    --terra-time-input-meridiem-button-border-color: #6f7477;
+    --terra-time-input-meridiem-button-color: #000;
+    --terra-time-input-meridiem-button-hover-background-color: #e5e5e5;
+    --terra-time-input-meridiem-button-hover-border-color: #6f7477;
+    --terra-time-input-meridiem-button-hover-color: #000;
+    --terra-time-input-meridiem-button-padding-bottom: 0.25rem;
+    --terra-time-input-meridiem-button-padding-top: 0.25rem;
+    --terra-time-input-meridiem-button-selected-background-color: #595f62;
+    --terra-time-input-meridiem-button-selected-border-color: #6f7477;
+    --terra-time-input-meridiem-button-selected-color: #fff;
+    --terra-time-input-meridiem-button-selected-hover-background-color: #595f62;
+    --terra-time-input-meridiem-button-selected-hover-border-color: #6f7477;
+    --terra-time-input-meridiem-button-selected-hover-color: #fff;
+
+    // Disabled
+    --terra-time-input-disabled-background-color: #f4f4f4;
+    --terra-time-input-disabled-border: 1px solid #b2b5b6;
+    --terra-time-input-disabled-color: #9b9fa1;
+    --terra-time-input-disabled-opacity: 1;
+    
+    // Invalid
+    --terra-time-input-invalid-background-color: #fff;
+    --terra-time-input-invalid-border: 1px solid #dedfe0;
+    --terra-time-input-invalid-box-shadow: none;
+
+    // Invalid - Hover
+    --terra-time-input-invalid-hover-border: 1px solid #dedfe0;
+    --terra-time-input-invalid-hover-box-shadow: none;
+
+    // Invalid - Focus
+    --terra-time-input-invalid-focus-border: 1px solid #4cb2e9;
+    --terra-time-input-invalid-focus-box-shadow: 0 0 8px #a6d9f4;
+
+    // Invalid - Disabled
+    --terra-time-input-invalid-disabled-background-color: #f4f4f4;
+    --terra-time-input-invalid-disabled-border: 1px solid #b2b5b6;
+    --terra-time-input-invalid-disabled-color: #9b9fa1;
+    --terra-time-input-invalid-disabled-opacity: 1;
+
+    // Incomplete
+    --terra-time-input-incomplete-background-color: #fefd9a;
+    --terra-time-input-incomplete-hover-background-color: #fefd9a;
+    --terra-time-input-incomplete-focus-background-color: #fefd9a;
+    --terra-time-input-incomplete-focused-input-background-color: #b9d7e9;
+
+    // Incomplete - Disabled
+    --terra-time-input-incomplete-disabled-background-color: #f4f4f4;
+    --terra-time-input-incomplete-disabled-border: 1px solid #b2b5b6;
+    --terra-time-input-incomplete-disabled-color: #9b9fa1;
+    --terra-time-input-incomplete-disabled-opacity: 1;
+
+  
+    /* -------------------------------------------------------------------- 
+    * Values only needed for 4.14.0 and before
```

Comment:
Can I remove these variables as they have been removed from the latest version, or will it impact consumers (if any) using version below 4.14.0 ? cc @mjhenkes @benbcai

Reply:
Looks like there are still some projects locking time-input to versions before 4.14.0 but I don't know why we kept these around to begin with. We didn't do that for lowlight so I don't know why we're doing it for fusion. I hate to have to keep track of all of these variables that have already been removed in the latest version.

Reply:
Delete these

Reply:
Deleted 1bc859c